controllers/config/grafanaIni_test.go

```diff
@@ -221,6 +248,24 @@ role_attribute_strict = true
 scopes = readAPI
 token_url = https://gitlab.com/oauth/token
 
+[auth.jwt]
+allow_assign_grafana_admin = true
+auto_sign_up = true
+cache_ttl = 60m
+email_claim = sub
+enable_login_token = true
```

Comment:
Unable to find enable_login_token in https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/

Reply:
Hey @NissesSenap 
Same thing as my previous PR, I've found all the config here: https://github.com/grafana/grafana/blob/main/conf/sample.ini